packages/interface/src/components/interface-skeleton/index.js

```diff
@@ -182,30 +182,35 @@ function InterfaceSkeleton(
 								ariaLabel={ mergedLabels.secondarySidebar }
 								as={ motion.div }
 								initial="closed"
-								animate={
-									isMobileViewport ? 'mobileOpen' : 'open'
-								}
+								animate="open"
 								exit="closed"
 								variants={ {
 									open: { width: secondarySidebarSize.width },
 									closed: { width: 0 },
-									mobileOpen: { width: '100vw' },
 								} }
 								transition={ defaultTransition }
 							>
-								<div
+								<motion.div
 									style={ {
 										position: 'absolute',
 										width: isMobileViewport
 											? '100vw'
 											: 'fit-content',
 										height: '100%',
-										right: 0,
+										left: 0,
 									} }
+									initial="closed"
+									animate="open"
```

Comment:
Can we remove this so that the two components can use the same animation?

Reply:
Possibly.

Reply:
Yes we can, I did that in 2525a5f8cd so that now the parent controls the animation.